axelar-chains-config/info/stagenet.json

```diff
@@ -1677,6 +1677,124 @@
           "deployer": "0xBeF25f4733b9d451072416360609e5A4c115293E"
         }
       }
+    },
+    "sui": {
+      "name": "Sui",
+      "axelarId": "sui",
+      "networkType": "testnet",
+      "rpc": "https://sui-testnet-rpc.publicnode.com:443",
+      "tokenSymbol": "SUI",
+      "chainType": "sui",
+      "explorer": {
+        "name": "Suiscan",
+        "url": "https://suiscan.xyz/testnet"
+      },
+      "contracts": {
+        "AxelarGateway": {
+          "address": "0xa7ecca5ee73b60b70d35d94d7e4b553249b10cceabcd6eab9a1286869914359e",
+          "structs": {
+            "WeightedSigner": "0xa7ecca5ee73b60b70d35d94d7e4b553249b10cceabcd6eab9a1286869914359e::weighted_signer::WeightedSigner",
+            "Bytes32": "0xa7ecca5ee73b60b70d35d94d7e4b553249b10cceabcd6eab9a1286869914359e::bytes32::Bytes32",
+            "WeightedSigners": "0xa7ecca5ee73b60b70d35d94d7e4b553249b10cceabcd6eab9a1286869914359e::weighted_signers::WeightedSigners",
+            "Signature": "0xa7ecca5ee73b60b70d35d94d7e4b553249b10cceabcd6eab9a1286869914359e::proof::Signature",
+            "Proof": "0xa7ecca5ee73b60b70d35d94d7e4b553249b10cceabcd6eab9a1286869914359e::proof::Proof",
+            "Message": "0xa7ecca5ee73b60b70d35d94d7e4b553249b10cceabcd6eab9a1286869914359e::message::Message",
+            "SignersRotated": "0xa7ecca5ee73b60b70d35d94d7e4b553249b10cceabcd6eab9a1286869914359e::events::SignersRotated",
+            "ChannelCreated": "0xa7ecca5ee73b60b70d35d94d7e4b553249b10cceabcd6eab9a1286869914359e::events::ChannelCreated",
```

Comment:
Yes, it is there because if we add another type in a future release it will have a different address, and as we keep adding versions and potentially structs there will be many options. It looks unnecessary now that we have only one version but we are planning for the future.